### ISSUE TITLE
Ukraine not included in -E=EU

### ISSUE THREAD

**Issue by anbj** [GenericMappingTools/dcw-gmt#70]
No Ukraine:
```
$ gmt pscoast -E=EU+l | grep -i ukr
```

Ukraine not painted red:
```
$ gmt coast -EUA+gblack -E=EU+gred -W -png test -Na
```

**Comment by Esteban82** [comment]
Are you sure? What dcw version do you have?

**Comment by Esteban82** [comment]
I got this map with the last code.

![test](https://user-images.githubusercontent.com/17908951/158179599-229c21fc-4d91-43d7-aca0-de487e265efb.png)

**Comment by Esteban82** [comment]
It was fixed on DCW 2.0.1 .

**Comment by anbj** [comment]
*Sigh*, I'm sorry.

```
$ gmt --show-dcw
1.1.4
```

That's what happens you are forced to use your Windows computer.
